com.unity.perception/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 
+Renamed all instances of "KeyPoint" to "Keypoint" for standards adherence.
```

Comment:
@JonathanHUnity I was having a hard time coming up with a good log line here. Please feel free to change this!

Reply:
```suggestion
Renamed all types and members named "KeyPoint" to "Keypoint"
```
How about something like this?

Reply:
Also, we should add an upgrade note as well on changes they will have to make. It's going to be a pain for any current users.

Reply:
> How about something like this?

I like it. But technically, it is also other appearances of the term as part of other names, e.g. KeypointLabeler.
So I changed it to this:

Renamed all appearances of the term "KeyPoint" within types and names to "Keypoint".